pl_bolts/models/gans/pix2pix/components.py

```diff
@@ -0,0 +1,167 @@
+import torch
+from torch import nn
+
+
+def center_crop(image, new_shape):
+    h, w = image.shape[-2:]
+    n_h, n_w = new_shape[-2:]
+    cy, cx = int(h / 2), int(w / 2)
+    xmin, ymin = cx - n_w // 2, cy - n_h // 2
+    xmax, ymax = xmin + n_w, ymin + n_h
+    cropped_image = image[..., xmin:xmax, ymin:ymax]
+    return cropped_image
+
+
+class ConvBlock(nn.Module):
```

Comment:
I see that this block is really very frequently used. Conv + Bn + Act or Conv + Act + Bn.

Marking for #427 discussion.